data/campaigns/Eastern_Invasion/scenarios/15_Return_to_Wesnoth.cfg

```diff
@@ -0,0 +1,977 @@
+            scroll=no
+        [/message]
+        [delay]
+            time=250
+        [/delay]
+
+        {SCROLL_TO 5 35}
+        [animate_unit]
+            [filter]
+                side,canrecruit=5,yes
+            [/filter]
+            flag,hits=attack,no
+            [primary_attack]
+                range=ranged
+            [/primary_attack]
+            [facing]
+                x,y=4,34
+            [/facing]
+        [/animate_unit]
+        {GENERIC_UNIT 5 (Chocobone)      4 34} {ANIMATE}
+        [animate_unit]
+            [filter]
+                side,canrecruit=5,yes
+            [/filter]
+            flag,hits=attack,no
+            [primary_attack]
+                range=ranged
+            [/primary_attack]
+            [facing]
+                x,y=5,36
+            [/facing]
+        [/animate_unit]
+        {GENERIC_UNIT 5 (Skeleton Rider) 5 36} {ANIMATE}
+        [animate_unit]
+            [filter]
+                side,canrecruit=5,yes
+            [/filter]
+            flag,hits=attack,no
+            [primary_attack]
+                range=ranged
+            [/primary_attack]
+            [facing]
+                x,y=6,35
+            [/facing]
+        [/animate_unit]
+        {GENERIC_UNIT 5 (Bone Knight)    6 35} {ANIMATE}
+        [message]
+            speaker=Gweddry
+            message= _ "But look, the undead are laying siege and have built fortifications completely surrounding the city! The defenders must be grievously outnumbered."
+            scroll=no
+        [/message]
+        {SCROLL_TO 22 16}
+        [animate_unit]
+            [filter]
+                side,canrecruit=3,yes
+            [/filter]
+            flag,hits=attack,no
+            [primary_attack]
+                range=ranged
+            [/primary_attack]
+            [facing]
+                x,y=30,15
+            [/facing]
+        [/animate_unit]
+        {GENERIC_UNIT 3 (Revenant)   30 15} {ANIMATE}
+        [animate_unit]
+            [filter]
+                side,canrecruit=3,yes
+            [/filter]
+            flag,hits=attack,no
+            [primary_attack]
+                range=ranged
+            [/primary_attack]
+            [facing]
+                x,y=31,16
+            [/facing]
+        [/animate_unit]
+        {GENERIC_UNIT 3 (Banebow)    31 16} {ANIMATE}
+        [animate_unit]
+            [filter]
+                side,canrecruit=3,yes
+            [/filter]
+            flag,hits=attack,no
+            [primary_attack]
+                range=ranged
+            [/primary_attack]
+            [facing]
+                x,y=32,15
+            [/facing]
+        [/animate_unit]
+        {GENERIC_UNIT 3 (Deathblade) 32 15} {ANIMATE}
+        [message]
+            speaker=Dacyn
+            message= _ "Even this undead force appears to be (cough) but a small fraction of Mal-Ravanal’s greater host. (cough)(cough) I must meet with the king. Weldyn will fall if we do not act."
```

Comment:
Capitalise 'king'?

Reply:
I don't think it makes sense to capitalize "king" here, unless you add his name: "King Konrad".